pom.xml

```diff
@@ -31,8 +31,7 @@
 
```

Comment:
see `<version>` tag above this (~ line 28). To keep continuity with prior releases, and avoid "surprise", I think we should make this version 0.0.13 (instead of 1.0.0), which will become 0.0.14 at the next release.